Dockerfile

```
@@ -0,0 +1,21 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest
+
+WORKDIR /src
+RUN dnf install -y \
+    --setopt=deltarpm=0 \
+    --setopt=install_weak_deps=false \
+    --setopt=tsflags=nodocs \
+    httpd \
+    mod_ssl \
+    python3-mod_wsgi \
+    python3-pip\
+    && dnf clean all
```

Just FYI, I was trying to build this with ubi-minimal (with podman), but I get some cryptic error (`error: Error -1 running transaction`).

```suggestion
FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3

WORKDIR /src
RUN microdnf install \
    httpd \
    mod_ssl \
    python3-mod_wsgi \
    python3-pip\
    && microdnf clean all
```